docs/disk-caching/README.md

```diff
@@ -22,28 +27,49 @@ Disk caching can be enabled by updating the `cache` config settings for Minio se
 },
 ```
 
-To update the configuration, use `mc admin config get` command to get the current configuration file for the minio cluster in json format, and save it locally.
+Use `mc admin config get` to get the current JSON configuration file for the Minio deployment, and save it locally:
+
 ```sh
 $ mc admin config get myminio/ > /tmp/myconfig
```

Comment:
Should we add Windows direction as well?  Per @harshavardhana:

`mc cat C:\temp\myconfig.json | mc admin config set local_minio`